src/types.ts

```diff
@@ -8,13 +8,18 @@ export type ObserverOptions = {
   threshold?: number
 }
 
-export type Props = {
+type Props = {
   defaultImage: string
   image: string
   errorImage?: string
   onLoaded?: () => void
   options?: ObserverOptions
   style?: Object
-  height: number | string
-  width: number | string
+  height?: number | string
+  width?: number | string
```

Comment:
I feel that it is bad practice to skip `height` and `width` for images since it gives the user a bumpy rendering and since we are using a default image (that is static?) we should know the dimensions. Right?

Reply:
for background images I use className (or in some places styleName) instead of defining width/height for each single image.

Reply:
@Cordazar, sounds reasonable 👍